python-client/giskard/ml_worker/websocket/utils.py

```diff
@@ -309,7 +309,7 @@ def function_argument_to_ws(value: Dict[str, Any]):
         #         **self.parse_function_arguments(arg.args))
```

Comment:
can we clean this commented code up?

Reply:
Sure, there were such code before. I thought it was someone keeping it on purpose, in order to remind something.